docs/en/setup/service-agent/java-agent/How-to-enable-kafka-reporter.md

```diff
@@ -1,6 +1,6 @@
 # How to enable Kafka Reporter
 
-The Kafka reporter plugin support report traces, JVM metrics, Instance Properties, and profiled snapshots to Kafka cluster, which is disabled in default. Move the jar of the plugin from `optional-reporter-plugins` to `reporter-plugins` for activating.
+The Kafka reporter plugin support report traces, JVM metrics, Instance Properties, and profiled snapshots to Kafka cluster, which is disabled in default. Move the jar of the plugin, `kafka-reporter-plugin-x.y.z.jar`, from `agent/optional-reporter-plugins` to `agent/reporter-plugins` for activating.
```

Comment:
I thought the issues means no reporter-plugins folder? Do we scan that folder?

Reply:
I think we just need copy to plugins folder?

Reply:
Yeap, I fix it.